app/components/zitting/manage-zittingsdata.hbs

```diff
@@ -8,24 +30,19 @@
       <div class="au-o-flow">
         <div>
           <AuLabel>{{t "manageZittingsData.bestuursorganLabel"}}</AuLabel>
-          <PowerSelect
-            @placeholder={{t "manageZittingsData.selectPlaceholder"}}
-            @selected={{this.bestuursorgaan}}
-            @options={{this.bestuursorgaanOptions}}
-            @onChange={{this.changeSelect}}
-            as |bestuursorgaan|>
-              {{bestuursorgaan.isTijdsspecialisatieVan.naam}}
-              (periode: {{moment-format bestuursorgaan.bindingStart "MM/DD/YYYY"}} -
-              {{#if bestuursorgaan.bindingEinde}}{{moment-format bestuursorgaan.bindingEinde "MM/DD/YYYY"}}{{else}}nvt.{{/if}})
-          </PowerSelect>
+          {{this.bestuursorgaan.isTijdsspecialisatieVan.naam}}
```

Comment:
This is now simply displayed as plain text, which might not be the best idea. Should I show it as a disabled input field (or select) instead?

Reply:
I think we use both patterns at the moment, so for me it's ok.